src/styles/_header.scss

```diff
@@ -25,6 +25,12 @@
     color: #060b35;
     list-style: none;
     line-height: 60px;
+    transition: color 300ms;
+
+    &:hover {
+      color: #00acdc;
+      cursor: pointer;
```

Comment:
isn't it default option for <a>?

Reply:
Hi, @Bohdan-Maksymiuk, what do you mean? If you mean style for cursor: pointer, then I added this to have pointer over all nav link block, not only over link's text. It looks prettier for me.